lms/services/application_instance_getter.py

```diff
@@ -57,6 +57,18 @@ def lms_url(self):
         """
         return self._get_by_consumer_key().lms_url
 
+    def lms_host(self):
+        """
+        Return the hostname part of the current request's LMS URL.
+
+        :raise ConsumerKeyError: if the request's consumer key isn't in the DB
+        :raise ValueError: if the request's LMS URL can't be parsed
+
+        :return: the matching LMS host
+        :rtype: str
+        """
+        return self._get_by_consumer_key().lms_host()
```

Comment:
`ApplicationInstanceGetter` is a bit silly but I'm not fixing that in this PR

Reply:
~I feel like `ApplicationInstanceGetter` could do with giving you back an application instance for example rather than awkwardly wrapping every method of one, but yes, fodder for another day.~

I've noticed that we've also grown an "ApplicationInstanceService" which has a get method. A small tweak to that service would seem to make this makes this unnecessary. The `ApplicationInstanceService` requires a `consumer_key`, but if that was optional and a default of `request.lti_user.oauth_consumer_key` was present, then this could be:

`application_instance_service.get().lms_host()`

The same would be true for a lot of these straight pass through functions. I'm not sure what the plan is with the two services, so I'll assume there is one and it's in progress?